### PR TITLE
Updates on contribution process

### PR DESCRIPTION
**Pull request by jlevy**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -48,3 +53,19 @@ When you contribute, keep in mind these conventions:
...
+
+- Roles:
+  - **project leads:** General direction of process and overall quality of the Guide
+	- **editors:** Contributors own specific sections or aspects of the guide, usually by topic/section, reviewing PRs and contributing; requires expert knowledge
+	- **experts:** People with expert knowledge in various areas, who have agreed to review or help on demand with tougher questions or PRs
+	- **contributors:** Everyone who helps one way or another
+- All PRs are reviewed by an **editor** and for non-trivial changes, a **project lead**, usually in that order, but it can be reversed for expediency.
+- In addition, anyone with relevant knowledge is encouraged to review/comment on PRs
+- Both editors and project leads are responsible for checking for style or problems
+- Trivial changes may me merged in directly by project leads or editors
```
Comment (ThanosBaskous) [inline CONTRIBUTING.md]: "may be merged"

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -48,3 +53,19 @@ When you contribute, keep in mind these conventions:
   -	Note we try to make sections uniquely titled, so GitHub links to Markdown section anchors don’t collide and are stable.
 
 Note we keep consistent formatting in Markdown via [markdownfmt](https://github.com/shurcooL/markdownfmt). We run **admin/reformat.sh** to do this, but you don’t have to worry about it unless you really want to.
+
+### Editorial Process
+
+- Roles:
+  - **project leads:** General direction of process and overall quality of the Guide
```
Comment (ThanosBaskous) [inline CONTRIBUTING.md]: Consistency: bulleted items in other lists in this file end with periods.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -48,3 +53,19 @@ When you contribute, keep in mind these conventions:
...
+	- **experts:** People with expert knowledge in various areas, who have agreed to review or help on demand with tougher questions or PRs
+	- **contributors:** Everyone who helps one way or another
+- All PRs are reviewed by an **editor** and for non-trivial changes, a **project lead**, usually in that order, but it can be reversed for expediency.
+- In addition, anyone with relevant knowledge is encouraged to review/comment on PRs
+- Both editors and project leads are responsible for checking for style or problems
+- Trivial changes may me merged in directly by project leads or editors
+
+### Contact
+
+If you have concerns or additional ideas of ways to help, contact [@jlevy](https://github.com/jlevy) (e-mail or DM on Slack).
```
Comment (ThanosBaskous) [inline CONTRIBUTING.md]: Feel free to add me here as well.

File: README.md [hunk 4]
```diff
@@ -5,6 +5,8 @@ The Open Guide to Amazon Web Services
 
 [![Slack Chat](https://img.shields.io/badge/Chat-Slack-ff69b4.svg "Join us. Anyone is welcome!")](https://og-aws.slack.lexikon.io/) [![Ask a Question](https://img.shields.io/badge/%3f-Ask%20a%20Question-dc9d47.svg "Questions help improve the Guide")](https://airtable.com/shrXZ61VrovWfXYBg)
 
+[Credits](AUTHORS.md) ∙ [Contributing guidelines](CONTRIBUTING.md)
```
Comment (ThanosBaskous) [inline README.md]: Suggestion (consistency with how "Slack Chat" is capitalized above): "Contributing Guidelines"

**Comment by jlevy** [comment]
👍 Thanks, all addressed.

**Pull request merged**